packer/resources/features/ixgbevf/README.md

```diff
@@ -0,0 +1,10 @@
+Enhanced networking for Ubuntu Vivid
+====================================
+
+This builds and installs a more up to date version of the ixgbevf network driver
+which is used by many of the larger AWS instance types.
+
+See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html
+for more details as to why this is required.
+
```

Comment:
Maybe worth pointing out in the readme that this feature is pre-installed on all images.